README.md

```diff
@@ -190,6 +190,16 @@ npm run example
 
 The example will rebundle on change, but you have to refresh your browser to see changes (no hot reloading currently).
 
+### CORS Proxy
```

Comment:
can we specify that this is only for running the example?  eg: CORS_PROXY_URL env variable is useless when the webreader is being used by a different app